pkgs/games/hyperrogue/default.nix

```diff
@@ -16,6 +16,39 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ autoreconfHook SDL SDL_ttf SDL_gfx SDL_mixer libpng glew ];
 
+  desktopItem = makeDesktopItem {
+    name = "hyperrogue";
+    desktopName = "HyperRogue";
+    genericName = "HyperRogue";
+    comment = meta.description;
+    icon = "hyperrogue";
+    exec = "hyperrogue";
+    categories = "Game;AdventureGame;";
+  };
+
+  postInstall = ''
+    mkdir -p $out/share/applications
```

Comment:
Nit: these `cp + mv` combos could be replaced by single calls to `install`.

Reply:
Fair point. I've changed it to use `install` and it does look neater.